its/src/test/kotlin/org/sonarlint/intellij/its/tests/AllUiTests.kt

```diff
@@ -0,0 +1,358 @@
+import org.sonarlint.intellij.its.utils.OrchestratorUtils.Companion.executeBuildWithMaven
+import org.sonarlint.intellij.its.utils.OrchestratorUtils.Companion.executeBuildWithSonarScanner
+import org.sonarlint.intellij.its.utils.OrchestratorUtils.Companion.generateToken
+import org.sonarlint.intellij.its.utils.OrchestratorUtils.Companion.newAdminWsClientWithUser
+import org.sonarlint.intellij.its.utils.ProjectBindingUtils.Companion.bindProjectAndModuleInFileSettings
+import org.sonarlint.intellij.its.utils.SettingsUtils.Companion.clearConnections
+import org.sonarlint.intellij.its.utils.SettingsUtils.Companion.clearConnectionsAndAddSonarQubeConnection
+import org.sonarlint.intellij.its.utils.SettingsUtils.Companion.clickPowerSaveMode
+import org.sonarqube.ws.client.WsClient
+import org.sonarqube.ws.client.issues.DoTransitionRequest
+import org.sonarqube.ws.client.issues.SearchRequest
+import org.sonarqube.ws.client.settings.SetRequest
+
+@DisabledIf("isCLionOrGoLand")
+class AllUiTests : BaseUiTest() {
+
+    companion object {
+        @JvmField
+        @RegisterExtension
+        val ORCHESTRATOR: OrchestratorExtension = defaultBuilderEnv()
+            .setEdition(Edition.DEVELOPER)
+            .activateLicense()
+            .addBundledPluginToKeep("sonar-java")
+            .addBundledPluginToKeep("sonar-security")
+            .addBundledPluginToKeep("sonar-scala")
+            .addBundledPluginToKeep("sonar-php")
+            .addBundledPluginToKeep("sonar-python")
+            .build()
+
+        private lateinit var adminWsClient: WsClient
+
+        const val TAINT_VULNERABILITY_PROJECT_KEY = "sample-java-taint-vulnerability"
+        const val SECURITY_HOTSPOT_PROJECT_KEY = "sample-java-hotspot"
+        const val PROJECT_KEY = "sample-scala"
+        const val MODULE_PROJECT_KEY = "sample-scala-mod"
+        const val ISSUE_PROJECT_KEY = "sample-java-issues"
+
+        private var firstHotspotKey: String? = null
+        lateinit var token: String
+
+        @Throws(InvalidProtocolBufferException::class)
+        private fun getFirstHotspotKey(client: WsClient): String? {
+            val searchRequest = org.sonarqube.ws.client.hotspots.SearchRequest()
+            searchRequest.projectKey = SECURITY_HOTSPOT_PROJECT_KEY
+            val searchResults = client.hotspots().search(searchRequest)
+            val hotspot = searchResults.hotspotsList[0]
+            return hotspot.key
+        }
+
+        @JvmStatic
+        @BeforeAll
+        fun createSonarLintUser() {
+            adminWsClient = newAdminWsClientWithUser(ORCHESTRATOR.server)
+            token = generateToken(adminWsClient, "sonarlintUser")
+
+            clearConnectionsAndAddSonarQubeConnection(ORCHESTRATOR.server.url, token)
+        }
+    }
+
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    @DisabledIf("isCLionOrGoLand", disabledReason = "No Java security hotspots in CLion or GoLand")
```

Comment:
There is a `DisabledIf` at the class level, do we need one on each nested class?